### ISSUE TITLE
📎 Biome should report the location of the error in an invalid configuration file

### ISSUE THREAD

**Issue by Conaclos** [biomejs/biome#1627]
### Description

When Biome encounters a malformed configuration, it errors without giving the location of the error. For large configuration file, this can be hard to locate the error. Biome should report the location of the error.

```sh
❯ cat biome.json 
{
    linter: {}
}

❯ npx biome lint index.js 
/playroom/prettier/biome.json parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Property key must be double quoted

configuration ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Biome exited because the configuration resulted in errors. Please fix them.
```
